### DISCUSSION TITLE
腾龙工作室 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#98] [模板讨论]
腾龙工作室 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
签到结果：【今日签到获得"credit":"5"积分】

格式可以优化一下

**Reply by wjf0214** [reply]
已优化。

**Comment by miaoxiaozhao** [comment]
2024-06-15 18:37:12
日志：Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.tenlonstudio.com/wp-json/jwt-auth/v1/token

**Comment by Misuzu7** [comment]
这个不在维护了吗
